core/src/test/php/net/xp_framework/unittest/core/NamespaceAliasTest.class.php

```diff
@@ -44,19 +45,32 @@ public function defined_interface_exists() {
 
   #[@test]
   public function autoloaded_class_exists() {
-    new \net\xp_framework\unittest\core\NamespaceAliasAutoloadedFixture();    // Triggers autoloader
+    new NamespaceAliasAutoloadedFixture();              // Triggers autoloader
     $this->assertTrue(class_exists('net\xp_framework\unittest\core\NamespaceAliasAutoloadedFixture', false));
   }
 
   #[@test]
   public function autoloaded_namespaced_class_exists() {
-    new \net\xp_framework\unittest\core\NamespaceAliasAutoloadedNamespacedFixture();    // Triggers autoloader
+    new NamespaceAliasAutoloadedNamespacedFixture();    // Triggers autoloader
     $this->assertTrue(class_exists('net\xp_framework\unittest\core\NamespaceAliasAutoloadedNamespacedFixture', false));
   }
 
   #[@test]
   public function autoloaded_fq_class_exists() {
-    new \net\xp_framework\unittest\core\NamespaceAliasAutoloadedFQFixture();    // Triggers autoloader
+    new NamespaceAliasAutoloadedFQFixture();            // Triggers autoloader
     $this->assertTrue(class_exists('net\xp_framework\unittest\core\NamespaceAliasAutoloadedFQFixture', false));
   }
+
+  #[@test]
+  public function namespaced_classes_aliased_to_short_name_when_loaded_via_uses() {
+    uses('net.xp_framework.unittest.core.NamespaceAliasAutoloadedNamespacedFixture');
+    $this->assertTrue(class_exists('NamespaceAliasAutoloadedNamespacedFixture', false));
+  }
+
+  #[@test]
+  public function namespaced_classes_not_aliased_twice_when_loaded_via_uses() {
```

Comment:
Not sure what this test actually covers? Wouldn't you need two classes of different namespaces but with same "short name" to make your point?